Meta/check-style.py

```diff
@@ -44,8 +44,6 @@ def should_check_file(filename):
         return False
     if filename.startswith('Base/'):
         return False
-    if filename == 'Kernel/FileSystem/ext2_fs.h':
```

Comment:
and keep the exceptions in Meta, unless we go full read-spec-only redefintion of the ext2 constants and structures.